### ISSUE TITLE
Screen readers do not announce anything when Datalist autocomplete results appear

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#4379]
## Describe the bug

[Test URL]
https://clarity.design/documentation/datalist

[WCAG 2.1 Guideline]
Success Criterion 4.1.2 Name, Role, Value
(Level A)
https://clarity.design/documentation/datalist

[OS]
macOS version 10.15.1, Windows 10

[User Agents/Assistive Technologies]
macOS Safari/VoiceOver, Windows 10/NVDA/Firefox, Windows 10/NVDA/Edge

[Component]
Datalist form input

[Description]
Screen readers do not announce anything when Datalist autocomplete results appear. 

See the test results for the datalist element at http://cx-guidelines.vmware.com/examples/datalist.html.

[User Impact]
Screen reader users will not hear anything when they use the autocomplete capabilities of this control.

[Steps to Reproduce]
Using one of the listed screen reader/browser combinations, activate the autocomplete feature of the datalist and observe that you do not hear anything.

[Recommendation]  
Implement a control based on one of the following approaches.

- http://whatsock.com/tsg/Coding%20Arena/ARIA%20Comboboxes/ARIA%20Comboboxes%20(Native%20Inputs,%20Editable%20with%20Word%20Match)/demo.htm
 
- https://adamsilver.io/articles/building-an-accessible-autocomplete-control/

**Comment by gnomeontherun** [comment]
This seems to be a known bug with some browsers screen reader combinations.

See https://a11ysupport.io/tech/html/datalist_element.

I'm not sure there is anything actionable here, other than to use the Combobox when it is available if this is a problem for applications. I'm going to close this as unactionable due to it being out of our hands.
